### DISCUSSION TITLE
make: *** [Util/BuildTools/Linux.mk:137: setup] Error 1

### DISCUSSION THREAD

**Discussion by SirMichcik** [carla-simulator/carla#8118] [Q&A]
Hi! I have encountered an error while trying to make launch in carla root folder, here are my specs:

Ubuntu 20.04
Carla 0.9.13
UnrealEngine_4.26

I was using linux build guide, fixed some errors on the way and now im stuck on this:

cstring:77:9: error: no member named 'strcmp' in the global namespace; did you mean 'strchr'?
using ::strcmp;
cstring:76:9: error: no member named 'memcmp' in the global namespace; did you mean 'memchr'?
using ::memcmp;
cstring:74:9: error: no member named 'strcat' in the global namespace; did you mean 'strchr'?
using ::strcat;
cstring:73:9: error: no member named 'strncpy' in the global namespace
using ::strncpy;
 
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
ninja: build stopped: subcommand failed.
make: *** [Util/BuildTools/Linux.mk:137: setup] Error 1

There were of course 20 error messages, all about namespaces like this, i think it has something to do with compiler even though I am using clang 8 as requested.

**Comment by mohammadcp147** [comment]
Same Problem Here
 Need one help. I am stuck at the last stage of the video where we run     {make PythonAPI} it is throwing an error showing :- make:*** [Util/BuildTools/Linux.mk:142: setup] Error 1
